daprdocs/content/en/js-sdk-docs/js-server/_index.md

```diff
@@ -131,6 +159,47 @@ npm run start:dapr-grpc
 
 > ℹ️ **Note:** The `app-port` is required here, as this is where our server will need to bind to. Dapr will check for the application to bind to this port, before finishing start-up.
 
+## Communication Protocol for DaprClient
```

Comment:
@shubham1172 : Please see this section for preferential order for Dapr Client Communication Protocol

Reply:
Do we really need a full section for this? Can we just add comments when we initialize for the first time? Let me add it as a suggestion.